pkg/devfile/adapters/docker/utils/utils.go

```diff
@@ -66,3 +78,66 @@ func GetProjectVolumeLabels(componentName string) map[string]string {
 	}
 	return volumeLabels
 }
+
+// GetSupervisordVolumeLabels returns the label selectors used to retrieve the supervisord volume
+func GetSupervisordVolumeLabels() map[string]string {
+	supervisordLabels := map[string]string{
+		"name": adaptersCommon.SupervisordVolumeName,
```

Comment:
Since there be multiple versions of supervisord, could we make one of the labels the version of supervisord? So that old versions of the supervisord volume aren't accidentally used by odo